lib/custom/src/MShop/Service/Provider/Payment/Stripe.php

```diff
@@ -21,6 +22,101 @@ class Stripe
 	extends \Aimeos\MShop\Service\Provider\Payment\OmniPay
 	implements \Aimeos\MShop\Service\Provider\Payment\Iface
 {
+
+	private $beConfig = array(
+		'type' => array(
```

Comment:
'type' and 'onsite' are not required because they are fixed for Stripe

Reply:
Cool. Removed